src/Map/CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 ## Unreleased
 
--   Rename `render_map` Twig function `ux_map`
+-   Add `<twig:ux:map />` Twig component
+-   Update `ux_map` Twig function to accept an array 
 -   Deprecate `render_map` Twig function
+-   Rename `render_map` Twig function `ux_map`
```

Comment:
Do you think can we put:
```
-   Deprecate `render_map` Twig function
-   Rename `render_map` Twig function `ux_map`
```
in the first lines instead? 

It will be easier to understand the changelog when reading from top to bottom, so we can have something like:
```suggestion
-   Deprecate `render_map` Twig function
-   Rename `render_map` Twig function `ux_map
-   Add `<twig:ux:map />` Twig component
-   Update `ux_map` Twig function to accept an array 
```

Reply:
Also, it is `<twig:ux:map />` or `<twig:UX:Map />` (I see you used `UX:Map` in `twig.component` tag)? Or is it can insensitive?

Reply:
I think we use it some times "recent -> old" and some time in the other way so i really spent time to figure something.... and i hoped for some comment like this suggesting a direction haha :)

Reply:
I'v updated the CHANGELOG, tell me what you think, i tried to be the most DX-friendly possible

Reply:
> Also, it is <twig:ux:map /> or <twig:UX:Map /> (I see you used UX:Map in twig.component tag)? Or is it can insensitive?

Both `<twig:ux:icon` and now `<twig:ux:map `are indeed case-insensitive. This is something we need to sort now, i'm opening an issue to discuss it :)

Reply:
https://github.com/symfony/ux/issues/2118